lib/get-config.js

```diff
@@ -1,10 +1,10 @@
-import { dirname, extname } from "node:path";
+import { dirname } from "node:path";
 import { fileURLToPath } from "node:url";
 
 import { castArray, isNil, isPlainObject, isString, pickBy } from "lodash-es";
 import { readPackageUp } from "read-pkg-up";
 import { cosmiconfig } from "cosmiconfig";
-import resolveFrom from "resolve-from";
```

Comment:
Yes it is still being used in the `loadPlugin` function, and we can't replace it with `import-from-esm` it because doesn't support ESM named exports (it only returns the whole module [if the `default` export does not exist](https://github.com/sheerlox/import-from-esm/blob/844fe7a2a67a0a5a76ee8dc25f143acc45127516/index.js#L107)).
This part of the code is not subject to the Windows absolute path issue since the `import` statement explicitly specifies the `file://` protocol.

https://github.com/semantic-release/semantic-release/blob/d3aae0752c1f0719e0e6eff5c4d15cb2f90a4bfe/lib/plugins/utils.js#L50-L67